aim/sdk/sequence_collection.py

```diff
@@ -139,22 +140,45 @@ def __init__(
         self,
         repo: 'Repo',
         seq_cls=Sequence,
-        query: str = ''
+        query: str = '',
+        report_mode: int = 1,
```

Comment:
Define constants and replace numbers with them please.

Reply:
done ✅